web-ui/pages/index.vue

```diff
@@ -19,18 +19,33 @@ export default {
     EditorPanel,
     AppContainer,
   },
-  mounted() {
-    window.log = output => {
-      this.addToLog(output);
-    };
-  },
   data() {
     return {
       code:
-        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\n",
-      console: "",
+        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\nsdfsdf\n",
```

Comment:
Remove this invalid code - it is useful for testing but we don't really want the default script to have an error in it.